### PR TITLE
Into to stacks

### PR DESCRIPTION
**Pull request by DCamaron1**

### CODE REVIEW

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 1]
```diff
@@ -7,29 +7,32 @@
...
+		for (int i = 0; i < 100; i++) {
+			testStack.push(new Random().nextDouble() * 100);
+		}
+		String answer = JOptionPane.showInputDialog("Enter a number between 0 and 100 \n(you can include 0 and 100)");
+		String answer2 = JOptionPane.showInputDialog("Enter another number between 0 and 100");
+		int answerNum = Integer.parseInt(answer);
+		int answer2Num = Integer.parseInt(answer2);
+		int greaterNum;
+		int lesserNum;
+		if (answerNum > answer2Num) {
```
Comment (IsisJ) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: Good way for checking your values 👍

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 2]
```diff
@@ -7,29 +7,32 @@
...
+			greaterNum = answerNum;
+			lesserNum = answer2Num;
+		} else {
+			greaterNum = answer2Num;
+			lesserNum = answerNum;
+		}
+		System.out.println("Popping elements off stack... \nElements between " + lesserNum + " and " + greaterNum);
+		double currentValue = 0;
```
Comment (IsisJ) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: Do you want to keep the "\n" in line 28❓

File: src/_03_IntroToStacks/_01_IntroToStack.java [hunk 3]
```diff
@@ -7,29 +7,32 @@
...
+		Stack<Double> testStack = new Stack<>();
+		for (int i = 0; i < 100; i++) {
+			testStack.push(new Random().nextDouble() * 100);
+		}
+		String answer = JOptionPane.showInputDialog("Enter a number between 0 and 100 \n(you can include 0 and 100)");
+		String answer2 = JOptionPane.showInputDialog("Enter another number between 0 and 100");
+		int answerNum = Integer.parseInt(answer);
+		int answer2Num = Integer.parseInt(answer2);
+		int greaterNum;
+		int lesserNum;
```
Comment (IsisJ) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: Maybe you could put the int variables, on lines 19 and 20, at the top of your class ☘️

File: src/_03_IntroToStacks/_01_IntroToStack.java (hunk 2 above)
Comment (IsisJ) [inline src/_03_IntroToStacks/_01_IntroToStack.java]: Is there a reason you declared the 'currentValue' variable here or should you move it to the for loop🌜

**Review by mwoguri — APPROVED** [review]
👍 Please merge!

**Pull request merged**